__tests__/user/user.mutation.test.js

```diff
@@ -645,3 +645,96 @@ describe('mutations', () => {
     expect(response.statusCode).toEqual(400);
   });
 });
+
+describe('User`s mutation restictions tests', () => {
+  let adminId;
+  let userToken;
+  let firstName;
+  let lastName;
+  let email;
+
+  beforeAll(() => {
+    firstName = 'Pepo';
+    lastName = 'Markelo';
+    email = 'example@gmail.com';
+    adminId = '9c031d62a3c4909b216e1d86';
+    userId = '5f43af8522155b08109e0304';
+    userToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZjQzYWY4NTIyMTU1YjA4MTA5ZTAzMDQiLCJlbWFpbCI6ImV4YW1wbGVAZ21haWwuY29tIiwiaWF0IjoxNTk4Mjk3MjExLCJleHAiOjE1OTkxNjEyMTF9.G0qIuYSb89KNnKy7A2QNTmF6xcsGPhtuodNm8yoUC1s';
```

Comment:
Please, don't hardcode token, you need to login user in `beforeAll` statement and take the token from the backend

Reply:
fixed, added login user & admin tests for preventing from unexpected bugs and responses